framework/src/bundle/BundleResourceContainer.cpp

```diff
@@ -179,6 +181,27 @@ void BundleResourceContainer::FindNodes(
   }
 }
 
+void BundleResourceContainer::InitMiniz()
+{
+  // Assume that the bundle had its meta-data linked into a data section.
+  // If this assumption is false, fall back to reading the meta-data in a
+  // less than optimal way, in terms of memory utilization.
+  std::shared_ptr<RawBundleResources> rawBundleResourceData;
+  try {
+    m_ObjFile = BundleObjFactory().CreateBundleFileObj(m_Location);
+    rawBundleResourceData = m_ObjFile->GetRawBundleResourceContainer();
+  }
+  catch (const std::exception&) {}
```

Comment:
Does it make sense to log exceptions here, or just let it propagate upward, as we do with zip init errors below?

Reply:
I don't want to propagate the exception since I do want to attempt to load the zip file using ```mz_zip_reader_init_file``` as a fallback strategy.

logging the exception using the internal logger makes sense. I'll do that.